sda-commons-client-jersey/src/test/java/org/sdase/commons/client/jersey/ApiClientTimeoutTest.java

```diff
@@ -47,6 +49,11 @@ class ApiClientTimeoutTest {
 
   private ClientTestApp app;
 
+  @BeforeAll
+  static void beforeAll() {
+    WireMock.configureFor("http", "localhost", WIRE.getPort());
```

Comment:
Do we need a comment here? At least I don't understand why this configuration is needed.

Reply:
done